uds/database/data_record/text_data_record.py

```diff
@@ -0,0 +1,73 @@
+"""Definition of TextDataRecord which is class for encode and decode values of data records."""
+
+__all__ = ["TextTableDataRecord"]
+
+from typing import Dict, Optional
+
+from .abstract_data_record import AbstractDataRecord, DataRecordPhysicalValueAlias, DecodedDataRecord
+from .raw_data_record import RawDataRecord
+
+
+class TextTableDataRecord(RawDataRecord, AbstractDataRecord):
+    """Implementation for Text Data Record."""
+
+    def __init__(self, name: str, length: int, mapping: Optional[Dict[int, str]] = None) -> None:
```

Comment:
Let's assume mapping is always there (no Optional and None as default value). If there is no mapping, then we jsut have RawDataRecord.